src/Arcus.Templates.AzureFunctions.Http/AzureFunction.cs

```diff
@@ -0,0 +1,214 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+using Arcus.WebApi.Logging.Correlation;
+using GuardNet;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Formatters;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Net.Http.Headers;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Linq;
+using JsonSerializer = Newtonsoft.Json.JsonSerializer;
+
+namespace Arcus.Templates.AzureFunctions.Http
+{
+    /// <summary>
+    /// Represents the base HTTP web API functionality for an Azure Function implementation.
+    /// </summary>
+    public abstract class AzureFunction
```

Comment:
Shouldn't we include HTTP in the name somehow? Here is a (stupid) suggestion.
```suggestion
    public abstract class HttpBasedAzureFunction
```

Reply:
Is good. We can use it for now.